### PR TITLE
setup-mobile-1

### PR DESCRIPTION
**Pull request by otmanTR**
In that branch following changes made;
-Add index.html file
-Add style.css file
-Add linters
-Add toolbar&headline sections in index.html file
-Add properties in style.css file
-Add images
-Fixed linter errors

### CODE REVIEW

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @otmanTR ,

Good job implementing most of the project requirements! 
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔
- Nice Design. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

### Optional Suggestions

- [ ] Please check the inline comments below.

**_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            <a class="social" href="#">
+              <img src="Images\kedi.png" alt="github"> </a>
+            </a>
+          </li>
+          <li>
+            <a class="social" href="#">
+              <img src="Images\vin.png" alt="medium"> </a>
+            </a>
+          </li>
+       </ul>
```
Comment (Lizdev-05) [inline index.html:61]: [OPTIONAL] Kindly remove the list styling that we have on your link and please follow the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Q5JsjWdTgCKq90yN-0)design template.
**Your Design**
image

**Figma Design**
image

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @otmanTR, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
- Correct GitHub Flow 👍

- Good commit history 💰

- All linters passed 🏅

- Personalised README file 👍

 ## Required Changes ♻️


_Check the comments under the review._

## Optional suggestions
- Kindly add the background images appropriately in order to meet up with the Figma design template. 
**Your Design**
image

**Figma Design**
image

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+   
+    <div >
+      <nav>
+        <a><img class="logo" src="Images\Logo Placeholder.png" alt="mylogo" /></a>
+        
+
+        <img class="icons" src="Images\Enabled.png" alt="threeline" />
+      </nav>
+    </div>
+  </header>
```
[the remark this answers is not in the document]
Comment (otmanTR) [reply index.html:22]: Thanks for your return. U added text and images, which one I use to use? if I do it like you wrote, images and logo text comes together.

**Review by Debas-31 — APPROVED** [review]
> ## Hi @otmanTR  :wave:, 

# Status: Approved :heavy_check_mark:  :heavy_check_mark:  :heavy_check_mark: 

Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- README :heavy_check_mark: 
- PR title and summary :heavy_check_mark: 
- Commit message :heavy_check_mark:


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**